js/screens/Login/LoginScreen.js

```diff
@@ -42,27 +41,22 @@ class LoginContainer extends Component {
   handleSubmit() {
     const { email, password } = this.props;
 
-    const validEmail = /^([\w-.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/;
     const errorMessage = {
```

Comment:
this error message can be in the helper function. If the input email is not valid that function should return this error.